docs/synapse/devopsguide.rst

```diff
@@ -981,6 +981,123 @@ this to add a user named ``foouser`` with the uid 1234::
     Successfully built 21a12f395462
     Successfully tagged vertexproject/synapse-aha:v2.113.0-foouser
 
+Configure Custom CA Certificates for HTTP Requests
+--------------------------------------------------
+
+The Cortex can be configured to use additional CA certificates when making HTTP
+requests via Storm. To do this, you need to provide the certificates (in DER or
+PEM format) to the Cortex in a directory on disk, and then configure the Cortex
+to look up that directory via the ``tls:ca:dir`` configuration option.
+
+The following Compose file shows an example using this option.
+
+::
+
+    services:
+      00.cortex:
+        user: "999"
+        image: vertexproject/synapse-cortex:v2.x.x
+        network_mode: host
+        restart: unless-stopped
+        volumes:
+            - ./storage:/vertex/storage
+        environment:
+            SYN_CORTEX_TLS_CA_DIR: /vertex/storage/tls-ca-certs
```

Comment:
Can we show mapping in a separate volume?
In some production cases, mapping multiple volumes under `/vertex/storage` may not be possible; so we may not want to imply that is the best way to do this configuration.